extensions/amp-list/amp-list.md

```diff
@@ -127,11 +127,17 @@ In `<amp-list>`, you can use the [`items`](#items-optional) attribute to render
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` element can contain an element with an `overflow` attribute. This element is shown if the AMP Runtime cannot resize the `<amp-list>` element as requested, which is the case if the list is inside the viewport. If the list is outside the viewport, it will be automatically expanded.
+Optionally, the `<amp-list>` element can contain an element with the `overflow` attribute. The AMP `overflow` attribute to resize AMP components that fetch content if the AMP framework is unable to size the component as requested. A child element of `amp-list` with the `overflow` attribute will appear if one of the following conditions are met:
```

Comment:
> A child element of `amp-list` with the `overflow` attribute will appear if one of the following conditions are met

I think there is one more subtlety here. You have to specify the overflow element in order for it to appear. If you don't provide an overflow element (an element with the `overflow` attribute as a child of `<amp-list>`), none will appear. It will only appear if one was provided. There might be a valid conversation to be had about whether or not we should provide a default overflow element and what that should look like, but for now we don't, so the default behavior is that the `<amp-list>` will be cut off.

Reply:
Ah, great catch - attempted to clarify!